src/components/featured-article/macro.spec.js

```diff
@@ -0,0 +1,133 @@
+/** @jest-environment jsdom */
+
+import * as cheerio from 'cheerio';
+import axe from '../../tests/helpers/axe';
+import { renderComponent } from '../../tests/helpers/rendering';
+import { EXAMPLE_FEATURED_ARTICLE_WITH_CHART, EXAMPLE_FEATURED_ARTICLE_WITH_IMAGE } from './_test-examples';
+
+describe('Macro: Featured Article', () => {
+    describe('GIVEN: Params: required', () => {
+        describe('WHEN: chart is provided and image is not', () => {
+            const $ = cheerio.load(renderComponent('featured-article', EXAMPLE_FEATURED_ARTICLE_WITH_CHART));
+
+            test('THEN: it passes accessibility checks', async () => {
+                const results = await axe($.html());
+                expect(results).toHaveNoViolations();
+            });
+
+            test('THEN: it displays the correct title text', () => {
+                expect($('.ons-featured__title').text()).toContain('Economic Trends 2024');
+            });
+
+            test('THEN: it links the title correctly', () => {
+                expect($('.ons-featured__title a').attr('href')).toBe('/economy/economic-trends-2024');
+            });
+
+            test('THEN: it renders the onsChart component', () => {
+                expect($('[data-highcharts-base-chart]').length).toBe(1);
+            });
+
+            test('THEN: the chart has the correct Id', () => {
+                expect($('[data-highcharts-base-chart]').attr('data-highcharts-id')).toBe('growth-chart');
+            });
+
+            test('THEN: the chart has the correct type', () => {
+                expect($('[data-highcharts-base-chart]').attr('data-highcharts-type')).toBe('line');
+            });
+        });
+
+        describe('WHEN: image is provided and chart is not', () => {
+            const $ = cheerio.load(renderComponent('featured-article', EXAMPLE_FEATURED_ARTICLE_WITH_IMAGE));
+
+            test('THEN: it passes accessibility checks', async () => {
+                const results = await axe($.html());
+                expect(results).toHaveNoViolations();
+            });
+
+            test('THEN: it displays the correct title text', () => {
+                expect($('.ons-featured__title').text()).toContain('Population Insights');
```

Comment:
```suggestion
                expect($('.ons-featured__title').text()).toBe('Population Insights');
```
Can we not use `toBe`?